evm_loader/lib/Cargo.toml

```diff
@@ -19,7 +19,8 @@ spl-token = { version = "~3.5", default_features = false, features = ["no-entryp
 spl-associated-token-account = { version = "~1.1", default_features = false, features = ["no-entrypoint"] }
 bs58 = "0.4.0"
 hex = "0.4.2"
-serde = "1.0.147"
+serde = "1.0.183"
```

Comment:
Let's jump to `serde = "1.0.186"` (anything above `serde = "1.0.183"`), because `serde = "1.0.183"` contains some precompiled macro binaries which could not be reproduced. They were removed in [`serde = "1.0.184"`](https://github.com/serde-rs/serde/releases/tag/v1.0.184).

Reply:
Done